storage/internal/benchmarks/utils.go

```diff
@@ -108,6 +109,44 @@ func generateRandomFile(path string, size int64) (string, error) {
 	return f.Name(), err
 }
 
+// fillDirectoryRandomly fills the directory with up to 1000 different files and
+// subdirectories. Files may be located in any of the subdirectories.
+// The number of bytes in the created files is returned.
+func fillDirectoryRandomly(dirPath string, size int64) (int64, error) {
+	currDirectoryLevel := 0
+	currPath := dirPath
+	currNumBytes := int64(0)
+
+	// directory contains a random number of objects =< 1000
+	for i := randomInt(1, 1000); i > 0; i-- {
+		// choose at random whether to create a directory or a file
```

Comment:
In response to your top-level question, I think a flatter structure (10:1 or more files to directories) seems potentially more realistic, and means that we won't spend as much time during the download phase creating directories).

Reply:
I agree (thanks for the input!), I think 10:1 would be around the minimum ratio that would look more realistic as a directory structure - but I don't have data to back it up so let's see what Frank says

Reply:
As discussed offline, the directory will not contain any subdirectories.